core/src/main/java/apoc/util/Util.java

```diff
@@ -779,18 +778,14 @@ public static String withMapping(Stream<String> columns, Function<String, String
         return with.isEmpty() ? with : " WITH "+with+" ";
     }
 
-    public static boolean isWriteableInstance(GraphDatabaseAPI db) {
+    public static boolean isWriteableInstance(GraphDatabaseService db) {
+        return isWriteableInstance(db, db.databaseName());
+    }
+    
+    public static boolean isWriteableInstance(GraphDatabaseService db, String dbName) {
```

Comment:
I'd suggest collapsing this into a single method if we are not using the one that takes two parameters anywhere else

Reply:
I've splitted it because we're using an `Util.isWriteableInstance(db)` in `TTLLifeCycle.java` too.